handlers/labels.go

```diff
@@ -0,0 +1,55 @@
+package handlers
+
+import (
+	"log"
+	"strconv"
+)
+
+const (
+	// initialReplicasCount how many replicas to start of creating for a function, this is
+	// also used as the default return value for getMinReplicaCount
+	initialReplicasCount = 1
+
+	// OFFunctionNameLabel is the label key used by OpenFaaS to store the function name
+	// on the resources managed by OpenFaaS for that function.  This key is also used to
+	// denote that a resource is a "Function"
+	OFFunctionNameLabel = "faas_function"
+	// OFFunctionMinReplicaCount is a label that user's can set and will be passed to Kubernetes
+	// as the Deployment replicas value.
+	OFFunctionMinReplicaCount = "com.openfaas.scale.min"
+)
+
+// parseLabels will copy the user request labels and ensure that any required internal labels
+// are set appropriately.
+func parseLabels(functionName string, requestLables *map[string]string) map[string]string {
```

Comment:
I'm not seeing any parsing in `parseLabels`. Is this the correct name for the function?

Reply:
Reasonable alternatives could be `buildLabels` or  `getLabels`